packages/purpose-process/src/model/domain/models.ts

```diff
@@ -24,3 +29,16 @@ export type ApiRiskAnalysisFormSeed = z.infer<
 export type ApiReversePurposeUpdateContent = z.infer<
   typeof api.schemas.ReversePurposeUpdateContent
 >;
+
+export type ApiGetPurposesFilters = {
+  name?: string;
+  eservicesIds: EServiceId[];
+  consumersIds: TenantId[];
+  producersIds: TenantId[];
+  states: PurposeVersionState[];
+  excludeDraft: boolean | undefined;
+};
```

Comment:
Could you remove this? Now we are using `GetPurposesFilters` that is [here](https://github.com/pagopa/interop-be-monorepo/pull/439/files#diff-3151b523d08fc3302f8957cb874cae400f744fa16160fab31d47d7a5c2854f3cR33)